nncf/experimental/torch/fx/transformations.py

```diff
@@ -116,12 +116,13 @@ def leaf_module_insertion_transformation(model: torch.fx.GraphModule):
     return leaf_module_insertion_transformation
 
 
-def bias_update_transformation_builder(node: NNCFNode, value: torch.Tensor) -> TransformationFNType:
+def bias_update_transformation_builder(node: NNCFNode, value: torch.Tensor, input_port_id: int) -> TransformationFNType:
```

Comment:
Why signature of function changed?

Reply:
To make it possible in future to use different port id to correct biases